browser/brave_wallet/brave_wallet_prefs_unittest.cc

```diff
@@ -274,4 +276,79 @@ TEST_F(BraveWalletPrefsUnitTest, MigrateShowTestNetworksToggle) {
   EXPECT_FALSE(GetPrefs()->HasPrefPath(kShowWalletTestNetworksDeprecated));
 }
 
+TEST_F(BraveWalletPrefsUnitTest, MigrateShowChainIdNetworkInfo) {
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetEthContractAddressMigrated, true);
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetsAddPreloadingNetworksMigrated,
+                         true);
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetsAddIsNFTMigrated, true);
+  GetPrefs()->SetBoolean(kBraveWalletEthereumTransactionsCoinTypeMigrated,
+                         true);
+  GetPrefs()->SetBoolean(kBraveWalletDeprecateEthereumTestNetworksMigrated,
+                         true);
+
+  EXPECT_FALSE(
+      GetPrefs()->HasPrefPath(kBraveWalletTransactionsChainIdMigrated));
+  EXPECT_FALSE(GetPrefs()->GetBoolean(kBraveWalletTransactionsChainIdMigrated));
+
+  const char ethTxId[] = "b1e8dda1";
+  const auto ethPath = base::StringPrintf("%s.%s", "mainnet", ethTxId);
```

Comment:
`base::Strcat({"mainnet", ".", ethTxId})` seems better? Or even `base::Strcat({"mainnet.", ethTxId})`

Reply:
changed it to base::JoinString, thanks